ports/geckolib/glue.rs

```diff
@@ -356,7 +356,7 @@ pub extern "C" fn Servo_StyleSet_Drop(data: *mut RawServoStyleSet) -> () {
 }
 
 pub struct GeckoDeclarationBlock {
-    pub declarations: Option<PropertyDeclarationBlock>,
+    pub declarations: Option<Arc<PropertyDeclarationBlock>>,
```

Comment:
I'm concerned about this one. `GeckoDeclarationBlock` is refcounted in the Gecko side, and I thought the whole point of removing `Arc` inside `PropertyDeclarationBlock` is to remove the double indirection that a refcounted object holding another refcounted object while they are expected to have the same lifetime.

Do you have plan to remove this `Arc` as well, or probably we'd better redisign `GeckoDeclarationBlock` so that we can refcount `PropertyDeclarationBlock` from the Gecko side directly? Or do we have to live with the double indirection if both changes are hard?